<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>391</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(5, 5, 5);</string>
    </property>
    <widget class="QLabel" name="label_Crypto">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>-10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
color: rgb(220,220,220);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#ffffff;&quot;&gt;Crypto:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Tempo_grafico">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
color: rgb(220,220,220);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#ffffff;&quot;&gt;Tempo Gráfico:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Crypto">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>BNBUSDT</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Tempo_grafico">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>4h</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_Status">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
  color: red;
}
</string>
     </property>
     <property name="title">
      <string>Status</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>151</width>
        <height>94</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QRadioButton" name="radioButton_vendido">
         <property name="styleSheet">
          <string notr="true">QRadioButton{
    color:                  white;
}
QRadioButton::indicator {
    width:                  14px;
    height:                 14px;
    border-radius:         7px;
}

QRadioButton::indicator:checked {
    background-color:       rgb(57,255,0);
    border:                 2px solid white;
}

QRadioButton::indicator:unchecked {
    background-color:       transparent;
    border:                 2px solid white;
}
</string>
         </property>
         <property name="text">
          <string>Vendido</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="radioButton_espera">
         <property name="styleSheet">
          <string notr="true">QRadioButton{
    color:                  white;
}
QRadioButton::indicator {
    width:                  14px;
    height:                 14px;
    border-radius:         7px;
}

QRadioButton::indicator:checked {
    background-color:       rgb(57,255,0);
    border:                 2px solid white;
}

QRadioButton::indicator:unchecked {
    background-color:       transparent;
    border:                 2px solid white;
}
</string>
         </property>
         <property name="text">
          <string>Em espera</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="radioButton_comprado">
         <property name="styleSheet">
          <string notr="true">QRadioButton{
    color:                  white;
}
QRadioButton::indicator {
    width:                  14px;
    height:                 14px;
    border-radius:         7px;
}

QRadioButton::indicator:checked {
    background-color:       rgb(57,255,0);
    border:                 2px solid white;
}

QRadioButton::indicator:unchecked {
    background-color:       transparent;
    border:                 2px solid white;
}
</string>
         </property>
         <property name="text">
          <string>Comprado</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_rentabilidade">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>211</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
  color: red;
}
</string>
     </property>
     <property name="title">
      <string>Rentabilidade %</string>
     </property>
     <widget class="QGraphicsView" name="graphicsView_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>191</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGraphicsView {
  color: transparent;
border: transparent
}
</string>
      </property>
     </widget>
     <widget class="QLabel" name="percentage">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
  color: red;
  background-color: transparent;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_recomendation">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>170</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
  color: red;
  background-color: transparent;
}
</string>
     </property>
     <property name="title">
      <string>Recomendação técnica</string>
     </property>
     <widget class="QLabel" name="recommendation">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: rgb(220,220,220);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBox_trade">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox{
    color:                  white;
}
</string>
     </property>
     <property name="text">
      <string>Trade</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="aviso_valor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
color: rgb(220,220,220);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_valor_de_caixa">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;Valor de Caixa $:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="valor_de_caixa">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>250</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_aviso">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>260</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
  color: red;
}
</string>
     </property>
     <property name="title">
      <string>Aviso</string>
     </property>
     <widget class="QLabel" name="aviso">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: rgb(220,220,220);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBox_sentimento">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>241</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox{
    color:                  white;
}
</string>
     </property>
     <property name="text">
      <string>sentimento do mercado</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_venda">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 25, 25);</string>
     </property>
     <property name="text">
      <string>Vender</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_compra">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>360</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(100, 250, 100);</string>
     </property>
     <property name="text">
      <string>Comprar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_quantidade">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;Quantidade:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="aviso_valor_quantidade">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
color: rgb(220,220,220);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <zorder>label_Crypto</zorder>
    <zorder>label_Tempo_grafico</zorder>
    <zorder>Crypto</zorder>
    <zorder>Tempo_grafico</zorder>
    <zorder>groupBox_Status</zorder>
    <zorder>groupBox_recomendation</zorder>
    <zorder>checkBox_trade</zorder>
    <zorder>groupBox_rentabilidade</zorder>
    <zorder>aviso_valor</zorder>
    <zorder>label_valor_de_caixa</zorder>
    <zorder>valor_de_caixa</zorder>
    <zorder>groupBox_aviso</zorder>
    <zorder>checkBox_sentimento</zorder>
    <zorder>pushButton_venda</zorder>
    <zorder>pushButton_compra</zorder>
    <zorder>label_quantidade</zorder>
    <zorder>aviso_valor_quantidade</zorder>
   </widget>
   <widget class="QLabel" name="aviso_valor_atual">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(220,220,220);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
